server/sonar-web/src/main/js/app/components/GlobalContainer.tsx

```diff
@@ -63,7 +62,7 @@ export default function GlobalContainer() {
                   </Workspace>
                 </BranchStatusContextProvider>
               </div>
-              <PromotionNotification />
+              {/* <PromotionNotification /> */}
```

Comment:
@jagadeshpaladugula Any reason to keep it in comments rather than removing the line in total?

Reply:
Nothing specific. I prefer commenting, rather than removing. If in case, if we need it, it is easy to co-releate.